packages/@rnw-scripts/beachball-config/lib-commonjs/beachball.config.d.ts

```diff
@@ -0,0 +1,7 @@
+/**
```

Comment:
Is there a reason these beachball config files are now popping up? Why are they new to the repo? Were we not checking them in before?

Reply:
Hmm I'm not sure. Looking at the 0.67-stable it looks like it also doesn't have the directory `beachball-config` maybe this directory isn't supposed to be in stable branches? I haven't seen any mention of that in the release notes, but maybe a nuance that forgot to be captured? @jonthysell